Sprint_4/source/foldButton_test.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask, request, render_template
+from CardGames import *
+
+app = Flask(__name__)
+
```

Comment:
Requirements are satisfied as you created an html button that triggers the fold! I don't see any logic error in calling the fold.html.

Reply:
Thanks for looking through and letting me know! It currently doesn't trigger any actual action.